include/xtt/util/valid_expiry.h

```diff
@@ -0,0 +1,42 @@
+/******************************************************************************
```

Comment:
Can we put the `is_expiry_passed` function into `include/xtt/certificates.h` instead of creating its own file? Since the main xtt code is using it, it would be nice to not have it in `util`.